### PR TITLE
PLT-2814 Fixed JavaScript errors upon paste

### PR DESCRIPTION
**Pull request by DavidLu1997**
Moved one line, should probably refactor out.

### CODE REVIEW

File: webapp/components/file_upload.jsx [hunk 1]
```diff
@@ -131,8 +131,6 @@ class FileUpload extends React.Component {
     }
 
     componentDidMount() {
-        var self = this;
-
```
Comment (hmhealey) [inline webapp/components/file_upload.jsx]: Removing this will break the drag and drop handling below

File: webapp/components/file_upload.jsx [hunk 2]
```diff
@@ -198,6 +196,7 @@ class FileUpload extends React.Component {
 
     pasteUpload(e) {
         var inputDiv = ReactDOM.findDOMNode(this.refs.input);
+        var self = this;
```
Comment (hmhealey) [inline webapp/components/file_upload.jsx]: You should just rename the instances of `self` below to `this`. It's not in an event handler so there's no change that `this` will be rebound and using `self` will also cause ESLint to print warnings

**Comment by jwilander** [comment]
+1

**Comment by coreyhulen** [comment]
+1

**Pull request merged**